migrations/versions/201608241100_2ce1756a2f12_add_abstract_tables.py

```diff
@@ -274,8 +274,14 @@ def upgrade():
         schema='events'
     )
 
+    op.alter_column('contributions', 'track_id', new_column_name='legacy_track_id', schema='events')
```

Comment:
Check dbdiff whether this (or the other revision) screws up any indexes - there's one on `event_id, track_id`.